expenses-manager-frontend/src/main/frontend/src/app/services/guard-service/guard.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { LocalService } from '../local-service/local.service';
+import { Router } from '@angular/router';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GuardService {
```

Comment:
Numele nu cred ca este foarte sugestiv. Este un guard sau un service? Cred ca un guard :) Ar trebui denumit AuthGuard.
Iar acest guard ar trebui sa implementeze CanActivate. See: [https://raghuvardhankaranam.medium.com/route-guards-in-angular-c2c01fe6167b]( https://raghuvardhankaranam.medium.com/route-guards-in-angular-c2c01fe6167b) si sa punem acest guard pe rutele care ne intereseaza sa se faca o verificare inainte de a redirecta catre ele:

```
 {
    path: 'home',
    component: HomeComponent,
    canActivate: [AuthGuard], // aplică guard-ul
  },
```